sys/include/net/eddystone.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_eddystone Eddystone Constants
+ * @ingroup     net
+ * @brief       General values defined by the BLE Eddystone beacon format
+ *
+ * See https://github.com/google/eddystone/blob/master/protocol-specification.md
+ * @{
+ *
+ * @file
+ * @brief       Constants defined by the Eddystone specification
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef NET_EDDYSTONE_H
+#define NET_EDDYSTONE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Collection of general Eddystone constants
+ * @{
+ */
+#define EDDYSTONE_NAMESPACE_LEN         (10U)
+#define EDDYSTONE_INSTANCE_LEN          (6U)
+/** @} */
+
+/**
+ * @name    URL scheme prefix values
+ * @{
+ */
+#define EDDYSTONE_URL_HTTP_WWW          (0x00)      /**< http://www.URL */
```

Comment:
use \` around these URLs here and below:

```C
/**< `http://www.URL` */
```

Doxygen interprets these as valid URLs (which leads to nowhere of course) with the current usage.